generators/app/templates/docs/BaseStyles.md

```diff
@@ -0,0 +1,71 @@
+## Base Styles
```

Comment:
We should have a link to  the [wolox-equalizer](https://github.com/Wolox/equalizer) repo, which in my opinion should have this doc. I already told @FrankIglesias and @SKOLZ that this is here, in case they want to add it to that repo. For the time being this documentation isn't there, so I guess it's fine to have it here.